install_config/configuring_sdn.adoc

```diff
@@ -180,16 +182,14 @@ networkConfig:
   serviceNetworkCIDR: 172.30.0.0/16
   vxlanPort: 4789 <1>
 ----
-<1> vxlanPort is the value used by the nodes for the VXLAN Port. It can be an integer between 1-65,535, if unspecified
-by the cluster administrator the default value is 4789.
+<1> Set to the value used by the nodes for the VXLAN Port. It can be an integer between 1-65,535. The default value is `4789`.
```

Comment:
Style says to omit the comma in 65,535

I'd say something like "The VXLAN port number that the nodes use. Enter an integer in the range 1-65535." but :man_shrugging: